metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/GlossaryTermAspect.pdl

```diff
@@ -6,8 +6,9 @@ import com.linkedin.common.Ownership
 import com.linkedin.common.Status
 import com.linkedin.common.BrowsePaths
 import com.linkedin.glossary.GlossaryRelatedTerms
+import com.linkedin.schema.SchemaMetadata
 
 /**
  * A union of all supported metadata aspects for a CorpUser
  */
-typeref GlossaryTermAspect = union[GlossaryTermKey, GlossaryTermInfo, Ownership, Status, BrowsePaths, GlossaryRelatedTerms]
\ No newline at end of file
+typeref GlossaryTermAspect = union[GlossaryTermKey, GlossaryTermInfo, Ownership, Status, BrowsePaths, GlossaryRelatedTerms, SchemaMetadata]
```

Comment:
can you revert this change? we no longer need to extend the GlossaryTermAspect pdl file

Reply:
updated